src/common/infra/ofga/mod.rs

```diff
@@ -265,7 +276,7 @@ pub async fn init() -> Result<(), anyhow::Error> {
             if tuples.is_empty() {
                 log::info!("No orgs to update to the openfga");
             } else {
-                log::debug!("tuples not empty: {:#?}", tuples);
+                // log::debug!("tuples not empty: {:#?}", tuples);
```

Comment:
Can this change be reverted. This will help to check the tuples that are written during the alert folder RBAC migrations and possible issues